src/components/grid-view/__tests__/GridViewSlider.test.js

```diff
@@ -2,9 +2,36 @@ import React from 'react';
 
 import GridViewSlider from '../GridViewSlider';
 
+const intl = {
+    formatMessage: jest.fn(),
+};
+
 describe('components/grid-view/GridViewSlider', () => {
     test('should render()', () => {
-        const wrapper = shallow(<GridViewSlider columnCount={4} maxColumnCount={7} onChange={() => {}} />);
+        const wrapper = mount(
```

Comment:
if possible could we keep the `shallow`? `mount` comes with more overhead that's not always needed

if we add `export { GridViewSlider as GridViewSliderBase };` in `GridViewSlider.js`, then I think we could still use shallow?

Reply:
Hmm I tried this in so many different forms and couldn't get it to work. I was wondering if maybe because it's a const and not a class it doesn't work in the same way?

Reply:
I'm thinking something like this:

```
// inside of GridViewSlider.js
export { GridViewSlider as GridViewSliderBase };


// inside of GridViewSlider.test.js
import { GridViewSliderBase as GridViewSlider } from '../GridViewSlider';
```

and I think you'll need to change the `const intl` on line 5 to something like:
```
const intl = {
    formatMessage: message => message.defaultMessage,
};
```